templates/company-board/contracts/CompanyBoardTemplate.sol

```diff
@@ -48,10 +44,10 @@ contract CompanyBoardTemplate is BaseTemplate {
         _ensureMiniMeFactoryIsValid(_miniMeFactory);
     }
 
-    function prepareInstance() public {
+    function prepareInstance(string _boardTokenName, string _boardTokenSymbol, string _shareTokenName, string _shareTokenSymbol) public {
```

Comment:
We shouldn't parameterize the board token, see https://github.com/aragon/dao-templates/issues/111#issuecomment-514400907

Reply:
Totally missed that comment. Thx for the reminder!
*Updated* so that CompanyBoardTemplate only has the shareToken's name and symbol as parameters.